packages/web-app-files/src/spaces/views/Spaces.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="oc-p-s">
+    <h2 class="" v-text="$gettext('Spaces')"></h2>
+    <div class="">
+      <span v-text="$gettext('Access all project related files on one place.')"></span>
+      <a href="#" v-text="$gettext('Learn more about spaces.')"></a>
+      <h3 v-text="$gettext('Your spaces')"></h3>
+      <hr class="uk-margin-bottom" />
+    </div>
+    <div class="spaces-list uk-container uk-container-expand uk-padding-remove">
+      <list-loader v-if="loadSpacesTask.isRunning" />
+      <template v-else>
+        <div
+          v-if="!!spaces.length"
+          class="
+            uk-grid-match uk-grid-column-small uk-grid-row-large uk-text-center uk-child-width-1-3@s
+          "
+          uk-grid
+        >
+          <a v-for="space in spaces" :key="space.id" href="#">
+            <span class="spaces-list-card oc-border uk-card uk-card-default">
+              <span class="uk-card-media-top oc-border-b">
+                <img
+                  :src="space.image ? space.image : defaultImg"
```

Comment:
Can be simplified to
```suggestion
                  :src="space.image || defaultImg"
```